src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CISShadowMonitor.java

```diff
@@ -273,11 +281,10 @@ private synchronized void processCISShadow(int version, Map<String, Object> desi
      * @param version CIS shadow version
      * @param desired CIS shadow <b>desired</b> state
      */
-    private void updateCISShadowReportedState(int version, Map<String, Object> desired) {
+    private void updateCISShadowReportedState(String version, Map<String, Object> desired) {
```

Comment:
This works, but is confusing. Let's just update this to:

```
private void updateCISShadowReportedState(Map<String, Object> reported)
```

Then when calling this, pass the desired state and leave a comment saying that reported should now match desired